hw02/index.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+// hw02: Loops, conditional constructions, primitive data types
+
+/* Task 1
+Напишіть код який виводить в консоль значення від 1 до 100, але замість чисел кратних 3 має виводитись `Лол`, замість значень кратних 5 - `Кек`, а замість значень кратних і 3, і 5 - `ЛолКек`
+*/
+// console.log('Task1  \n');
+console.log('Task 1:\n');
+
+const lol = 3;
+const kek = 5;
+
+// switch
+console.log('\nSwitch:');
+for (let i = 1; i <= 100; i++) {
+	switch (true) {
+		case i % lol == 0 && i % kek == 0:
```

Comment:
```suggestion
		case i % lol === 0 && i % kek === 0:
```
Ніколи не використовуйте не строге порівняння, потенційно може призвести до багів